app/Http/Controllers/Admin/DiagramController.php

```diff
@@ -286,6 +291,11 @@ public function editInfoUser(Request $request)
 
     public function editSeat(Request $request)
     {
+        $this->validate($request, [
+            'seat_id' => 'required',
+            'color' => 'required'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/DiagramController.php</br>**Line**: 296</br>Each line in an array declaration must end in a comma